test/TestApps/AppLifecycleTestApp - Copy/pch.h

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why do we have an "AppLifecycleTestApp - Copy"?

Reply:
This shouldn't be in the repo. Apparently cleaning the solution doesn't remove old builds. I based the new app off the AppLifeCycleTestApp which created AppLifecycleTestApp, which I built to make sure it compiled. I need to remove this from the build files.